src/v/pandaproxy/schema_registry/handlers.cc

```diff
@@ -155,9 +129,42 @@ std::invoke_result_t<F> get_or_load(server::request_t& rq, F f) {
     // Load latest writes and retry
     vlog(plog.debug, "get_or_load: refreshing schema store on missing item");
     co_await rq.service().writer().read_sync();
+    vlog(plog.debug, "get_or_load: <<read_sync");
     co_return co_await f();
 }
 
+ss::future<server::reply_t>
+put_config_subject(server::request_t rq, server::reply_t rp) {
+    parse_accept_header(rq, rp);
+    auto sub = parse::request_param<subject>(*rq.req, "subject");
+    auto config = ppj::rjson_parse(
+      rq.req->content.data(), put_config_handler<>{});
+    rq.req.reset();
+
+    // get_or_load because we might be modifying a subject we don't have
+    // in cache yet
+    co_await get_or_load(rq, [&rq, sub, config]() -> ss::future<bool> {
+        vlog(plog.debug, ">>write_config");
+        return rq.service().writer().write_config(sub, config.compat);
+        vlog(plog.debug, "<<write_config");
+    });
+
+    auto json_rslt = ppj::rjson_serialize(config);
+    rp.rep->write_body("json", json_rslt);
+    co_return rp;
+}
+
+ss::future<server::reply_t>
+get_schemas_types(server::request_t rq, server::reply_t rp) {
+    parse_accept_header(rq, rp);
+    rq.req.reset();
+
+    static const std::vector<std::string_view> schemas_types{"AVRO"};
+    auto json_rslt = ppj::rjson_serialize(schemas_types);
+    rp.rep->write_body("json", json_rslt);
+    return ss::make_ready_future<server::reply_t>(std::move(rp));
+}
+
```

Comment:
This would be easier to review if it didn't move around in the file.

Reply:
Sorry about that, the helper had been declared below the functions being changed -- this will get squashed into an earlier commit so that it's in the right place to begin with